### ISSUE TITLE
Add labels to issues

### ISSUE THREAD

**Issue by Narigo** [jscraftcamp/jscc-project-creator#1]
As far as I can tell from the current script, it does not read / save the labels of the issues yet. Can we get them as well, while scraping them? It should make it easier for us to create the sponsoring issues (and board) for that.

**Comment by davelosert** [comment]
As I could see, right now there are only two labels used: `sponsoring` and the label of the respective year (e.g. `jscc20`). So I thought we woulnd't need them when we write as we could simply statically add them. What do you think?

**Comment by Narigo** [comment]
I think it makes sense to label them or at least have a way to tell later, in which board we need to put them - we have one sponsoring board and one for general issues. Since I've copy/pasted the sponsoring issues and added them to the board manually for half an hour, I'd like to get this done automatically in the future :)

We should filter out `/jscc\d+/` from the labels, so we can add the correct year label later, but keeping "sponsoring", "event preparation" or similar would make sense to me

**Comment by davelosert** [comment]
I think I didn't express myself clearly: I totally agree that we should give labels to the issues - and the future script should definitly due this.

My idea was just - instead of infering them from the previous year, we give them statically within the code before executing the creation script.

The current code was basically just a unique operation for me to save some copy+pasting work to get the needed issues into the JSON format..

To make it a bit clrearer, I just pushed an update to the `issues.json`: https://github.com/jscraftcamp/jscc-project-creator/blob/master/input/issues.json

I now manually separated sponsoring and general tickets. My assumption is that the tickets there won't really changed and that those are almost all issues we will need every year.

Once I created the script to write them back to Github, I though of code something like:

```
const issues = require('./input/issues.json')
const generalLabel = 'jscc21'
const sponsoringLabel = 'sponsoring'

addIssues({issues: issues.general, labels: [generalLabel]});
addIssues({issues: issues.sponsoring, labels: [generalLabel, sponsorin]});
```
So the process is only semi-automatic. But since we only have to do it once a year, I guess that's fine. :)
